README.md

```diff
@@ -26,6 +26,20 @@ tools do not aim to provide an optimization platform or a higher-level compiler
 target; instead they aim for full fidelity and compliance with the spec (e.g.
 1:1 round-trips with no changes to instructions).
 
+## Installation
+You can install Wabt command line tools on Linux or macOS using Homebrew:
```

Comment:
How about something like: "wabt is available on many plafforms as prepackaged binaries.  For example if you use Homebrew you can use:  .... And you use an apt-based linux distribution you can use: ...`

Reply:
Copy updated [8ef4cea](https://github.com/WebAssembly/wabt/pull/2514/commits/8ef4cea63f2731c05943e953fce27911044ad5cd)